pkg/operator/oauth.go

```diff
@@ -62,6 +64,29 @@ func ApplyOAuth(cr *v1alpha1.Console, rt *routev1.Route) (*oauthv1.OAuthClient,
 	return authClient, authConfigSecret, nil
 }
 
+// Deletes the Console Auth Secret when the Console ManagementState is set to Removed
+func DeleteOAuthSecret(cr *v1alpha1.Console) error {
+	authConfigSecret := newOauthConfigSecret(cr, "")
+	return sdk.Delete(authConfigSecret)
+}
+
+// deletes secret & eliminates redirectUris
+func NeutralizeOAuthClient(cr *v1alpha1.Console) error {
+	authClient := defaultOAuthClient(cr)
+	if err := sdk.Get(authClient); err != nil {
+		return err
+	}
+	nilSecret := ""
```

Comment:
I would just leave the secret set since it is better than having a public OAuth client.  You could set it to a random value if you wanted to be extra paranoid.

Reply:
Random value is better than empty string?